src/main/scala/org/allenai/plugins/CoreDependencies.scala

```diff
@@ -79,7 +79,7 @@ trait CoreDependencies {
   val sprayCaching = sprayModule("caching")
 
   // Spray json (separate from Spray toolkit)
-  val sprayJson = "io.spray" %% "spray-json" % "1.2.6"
+  val sprayJson = "io.spray" %% "spray-json" % "1.3.1"
```

Comment:
The only change I've found is that `JsArray` now takes a `Vector` rather than a `List`. If it's for the sake of performance, sure, I guess, but that does seem a little specific.

Also, curse github for not showing this comment until after I hit the merge button.